dbms/src/Flash/Statistics/ExecutionSummaryHelper.cpp

```diff
@@ -40,6 +42,16 @@ void fillTiExecutionSummary(
     execution_summary->mutable_tiflash_network_summary()->set_inter_zone_send_bytes(current.inter_zone_send_bytes);
     execution_summary->mutable_tiflash_network_summary()->set_inter_zone_receive_bytes(
         current.inter_zone_receive_bytes);
+    if (local_task)
```

Comment:
What does locak_task mean?

Reply:
It means these execution summaries are not received through ExchangeReceiver from other MPPTasks.